app/services/art_service/appointment_engine.rb

```diff
@@ -313,6 +313,22 @@ def amounts_brought_to_clinic(patient, session_date)
       @amounts_brought_to_clinic[amount['drug_inventory_id'].to_i] = amount['value_numeric'].to_f rescue 0
     end
 
+    # include amounts brought from previous facility
+    amounts_brought_from_previous_facility = ActiveRecord::Base.connection.select_all <<-SQL
+      SELECT obs.*
+      FROM obs
```

Comment:
This will always be called regardless whether it's a third or 10th visit. To ensure we are not calling it all the time it would be best if we could separate this and have it as another function and should only be called when

1. Amounts brought to the clinic on the session day is not available
2. This is their first visit after a transfer in. Other visits after this will not make sense